modules/k8s/tanka/update-helm-chart.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+CHART_NAME="$1"
+
+helm repo list | grep -i mintel > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+  helm repo add mintel https://mintel.gihub.io/helm-charts > /dev/null 2>&1
+fi
+
+helm repo update > /dev/null 2>&1
+
+result=$(helm search repo ${CHART_NAME})
+
+latest_version=`echo $result | sed "s|.*${CHART_NAME} \([0-9\.]*\) .*|\1|"`
+
+sed -i "\|.*${CHART_NAME}.*|,+1 d" chartfile.yaml
+
+tk tool charts add $1@$latest_version
```

Comment:
I'm pretty sure this results in only adding a value to the `Chartfile.yaml` , which means you will end up with:

```
requires:
- chart: mintel/standard-application-stack
  version: 0.1.3-rc5
- chart: mintel/standard-application-stack
  version: 1.0.1
- chart: mintel/standard-application-stack
  version: 1.0.2
- chart: mintel/standard-application-stack
  version: 1.0.3
```

Reply:
You can manipulate this easy enough with `yq` (maybe without even needing `tk tool charts`)

https://github.com/mintel/build-harness-extensions/blob/main/modules/k8s/tanka/generate.sh#L46

Reply:
the line before deletes existing entries :)

Reply:
Would you mind doing with `yq` instead? Mostly because it's for yaml parsing so generally safer.

I'm not 100% , but that `sed` might assume the version is always 1 line below, which can change.